src/webapp/api.py

```diff
@@ -62,17 +63,17 @@ def hello():
     return "Yes, we're up", 200
 
 
-@app.route("/startGame")
-def start_game():
+@app.route("/startGame", defaults={"difficulty_id": 1})
```

Comment:
Setter også en default difficulty_id i FE, som kan sendes med startGame. Bedre at den tas imot i BE istedenfor å bli satt uavhengig, mtp feilhåndtering (huk tak i meg så kan jeg forklare hva jeg mener hehe;)).

Reply:
Fiksa nå